.evergreen/run-orchestration.sh

```diff
@@ -30,13 +31,23 @@ if [ "$SSL" != "nossl" ]; then
    ORCHESTRATION_FILE="${ORCHESTRATION_FILE}-ssl"
 fi
 
+if [ ! -z "$STORAGE_ENGINE" ]; then
+  ORCHESTRATION_FILE="$STORAGE_ENGINE"
+fi
+
+# Only wiredTiger supports enableMajorityReadConcern. This can be removed once
+# mongo-orchestration properly checks the storageEngine is compatible.
+if [ -z "$STORAGE_ENGINE" || "$STORAGE_ENGINE" = "wiredtiger" ]; then
+  ENABLE_MAJORITY_READ_CONCERN="--enable-majority-read-concern"
```

Comment:
In case we ever run with `set -u`, lets initialize it with empty value since its unconditionally used later